Meta Description Fallback

sareh opened the pull request:
Resolves #1430

When running Lighthouse for a page, `/news/articles/c85pqyj5m2ko`, there is an error about the lack of description text.

<img width="762" alt="Screen Shot lighthouse" src="https://user-images.githubusercontent.com/3028997/54521006-ea1dfd00-4961-11e9-812c-9ebfa3d3c94f.png">

After this fix, we do not have this error.

**Overall change:** When Ares does not provide a promo.summary to use in the meta description element, we use the value from seoHeadline. 

**Code changes:**

- Fallback value added
- Cypress test updated
- Lighthouse SEO threshold value returned back to 80. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

sareh committed:
Fallback for meta description to use seoHeadline when summary is not provided

ChrisBAshton commented:
Blocked on #1417, after which we should revert the Lighthouse threshold change introduced in that PR.

ChrisBAshton commented:
Related to #842?

sareh committed:
Merge branch 'latest' into metadata-description-fallback

sareh committed:
Merge branch 'latest' into metadata-description-fallback

sareh committed:
Cypress test for description should check for seoHeadline when summary is not provided

sareh committed:
SEO lighthouse threshold returned to 80

sareh commented:
@ChrisBAshton I created a bug ticket that specifically focuses on this issue: #1430

dr3 commented on package.json:
```diff
@@ -39,7 +39,7 @@
     "test:e2e:storybook:ci": "run-p --race storybook:ci cypress:storybook",
     "test:e2e:storybook:interactive": "npm run killStorybook && run-p --race storybook cypress:storybook:interactive",
-    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70",
+    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70",
```
```suggestion
    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80",
```

Commit:
Update package.json

ChrisBAshton reviewed (dismissed):
Nice

ChrisBAshton committed:
merge with latest

Conflict in package.json; we updated the Lighthouse SEO score, but 'latest' updated the Optimo IDs

ChrisBAshton committed:
Merge branch 'latest' into metadata-description-fallback

sareh committed:
Merge branch 'latest' into metadata-description-fallback
